scss/variables/core/_breakpoints.scss

```diff
@@ -7,94 +7,196 @@
 //
 
 //
-// $small-screen
+// $screen-xs
 //
-// Markup: $small-screen:  481px !default;
+// Markup: $screen-xs: 0px !default;
 //
-// Styleguide: Variables.Core.Breakpoints.small-screen
+// Styleguide: Variables.Core.Breakpoints.screen-xs
 //
-$small-screen: 481px !default;
+$screen-xs: 0 !default;
```

Comment:
Shouldn't this be the smallest breakpoint greater than 0?

Reply:
That's how I always do it, where 0 is baseline and xs is something like 320px -- so, I can easily change this.  I believe I did this based on how Figma was laid out.

Reply:
Ah, I just looked at the figma docs again. xs is set to 0. So there really is only 4 breakpoints. This is ok to keep it at 0. I suspect the `mq` will be for the range between 0 and grid-sm.